modules/foundational.cft.yaml

```diff
@@ -0,0 +1,234 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Sysdig Secure Onboarding
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+    - Label:
+        default: Sysdig Assigned Settings (Do not change)
+      Parameters:
+      - NameSuffix
+      - ExternalID
+      - TrustedIdentity
+      - IsOrganizational
+      - OrganizationalUnitIDs
+    ParameterLabels:
+      NameSuffix:
+        default: Name Suffix
+      ExternalID:
+        default: External ID
+      TrustedIdentity:
+        default: Trusted Identity
+      IsOrganizational:
+        default: Is Organizational
+      OrganizationalUnitIDs:
+        default: Organizational Unit IDs
+Parameters:
+  NameSuffix:
+    Type: String
+    Description: Suffix to append to the resource name identifiers
+    AllowedPattern: '[0-9a-z]+'
+    MaxLength: 8
+    MinLength: 4
+  ExternalID:
+    Type: String
+    Description: Sysdig assigned token that proves you own this account
+  TrustedIdentity:
+    Type: String
+    Description: The Role in Sysdig's AWS Account with permissions to your account
+  IsOrganizational:
+    Type: String
+    Description: Install into an organization
+    Default: 'false'
+    AllowedValues:
+    - 'true'
+    - 'false'
+  OrganizationalUnitIDs:
+    Type: CommaDelimitedList
+    Description: Comma separated list of organizational unit IDs to deploy
+Conditions:
+  IsOrganizational:
+    Fn::Equals:
+    - Ref: IsOrganizational
+    - 'true'
+Resources:
+  ConfigPostureRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub sysdig-secure-${NameSuffix}
```

Comment:
Can we change this role name to `sysdig-secure-posture-${NameSuffix}` to keep it consistent?

Reply:
updated